### PR TITLE
urllib3 requirements fix

### LINKED ISSUE #829: 9.0.0 install package versions conflict
Hello, I cannot install client because version clashes.

Long story short: choosen urllib3 is too new for requests package.

Full log:

```
docker run --rm -it python pip install kubernetes --no-color --progress-bar off

Collecting kubernetes
  Downloading https://files.pythonhosted.org/packages/00/f7/4f196c55f1c2713d3edc8252c4b45326306eef4dc10048f13916fe446e2b/kubernetes-9.0.0-py2.py3-none-any.whl (1.4MB)

Collecting six>=1.9.0 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Collecting python-dateutil>=2.5.3 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/41/17/c62faccbfbd163c7f57f3844689e3a78bae1f403648a6afb1d0866d87fbb/python_dateutil-2.8.0-py2.py3-none-any.whl (226kB)

Collecting requests (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/7d/e3/20f3d364d6c8e5d2353c72a67778eb189176f08e873c9900e10c0287b84b/requests-2.21.0-py2.py3-none-any.whl (57kB)

Collecting google-auth>=1.0.1 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/c5/9b/ed0516cc1f7609fb0217e3057ff4f0f9f3e3ce79a369c6af4a6c5ca25664/google_auth-1.6.3-py2.py3-none-any.whl (73kB)

Collecting urllib3>=1.23 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/39/ec/d93dfc69617a028915df914339ef66936ea976ef24fa62940fd86ba0326e/urllib3-1.25.2-py2.py3-none-any.whl (150kB)

Collecting pyyaml>=3.12 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/9f/2c/9417b5c774792634834e730932745bc09a7d36754ca00acf1ccd1ac2594d/PyYAML-5.1.tar.gz (274kB)

Collecting websocket-client!=0.40.0,!=0.41.*,!=0.42.*,>=0.32.0 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/29/19/44753eab1fdb50770ac69605527e8859468f3c0fd7dc5a76dd9c4dbd7906/websocket_client-0.56.0-py2.py3-none-any.whl (200kB)

Requirement already satisfied: setuptools>=21.0.0 in /usr/local/lib/python3.7/site-packages (from kubernetes) (41.0.1)
Collecting requests-oauthlib (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/c2/e2/9fd03d55ffb70fe51f587f20bcf407a6927eb121de86928b34d162f0b1ac/requests_oauthlib-1.2.0-py2.py3-none-any.whl
Collecting certifi>=14.05.14 (from kubernetes)
  Downloading https://files.pythonhosted.org/packages/60/75/f692a584e85b7eaba0e03827b3d51f45f571c2e793dd731e598828d380aa/certifi-2019.3.9-py2.py3-none-any.whl (158kB)

Collecting chardet<3.1.0,>=3.0.2 (from requests->kubernetes)
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)

Collecting idna<2.9,>=2.5 (from requests->kubernetes)
  Downloading https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl (58kB)

Collecting cachetools>=2.0.0 (from google-auth>=1.0.1->kubernetes)
  Downloading https://files.pythonhosted.org/packages/39/2b/d87fc2369242bd743883232c463f28205902b8579cb68dcf5b11eee1652f/cachetools-3.1.0-py2.py3-none-any.whl
Collecting pyasn1-modules>=0.2.1 (from google-auth>=1.0.1->kubernetes)
  Downloading https://files.pythonhosted.org/packages/91/f0/b03e00ce9fddf4827c42df1c3ce10c74eadebfb706231e8d6d1c356a4062/pyasn1_modules-0.2.5-py2.py3-none-any.whl (74kB)

Collecting rsa>=3.1.4 (from google-auth>=1.0.1->kubernetes)
  Downloading https://files.pythonhosted.org/packages/02/e5/38518af393f7c214357079ce67a317307936896e961e35450b70fad2a9cf/rsa-4.0-py2.py3-none-any.whl
Collecting oauthlib>=3.0.0 (from requests-oauthlib->kubernetes)
  Downloading https://files.pythonhosted.org/packages/16/95/699466b05b72b94a41f662dc9edf87fda4289e3602ecd42d27fcaddf7b56/oauthlib-3.0.1-py2.py3-none-any.whl (142kB)

Collecting pyasn1<0.5.0,>=0.4.1 (from pyasn1-modules>=0.2.1->google-auth>=1.0.1->kubernetes)
  Downloading https://files.pythonhosted.org/packages/7b/7c/c9386b82a25115cccf1903441bba3cbadcfae7b678a20167347fa8ded34c/pyasn1-0.4.5-py2.py3-none-any.whl (73kB)

Building wheels for collected packages: pyyaml
  Building wheel for pyyaml (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/ad/56/bc/1522f864feb2a358ea6f1a92b4798d69ac783a28e80567a18b
Successfully built pyyaml
ERROR: requests 2.21.0 has requirement urllib3<1.25,>=1.21.1, but you'll have urllib3 1.25.2 which is incompatible.
Installing collected packages: six, python-dateutil, urllib3, chardet, idna, certifi, requests, cachetools, pyasn1, pyasn1-modules, rsa, google-auth, pyyaml, websocket-client, oauthlib, requests-oauthlib, kubernetes
Successfully installed cachetools-3.1.0 certifi-2019.3.9 chardet-3.0.4 google-auth-1.6.3 idna-2.8 kubernetes-9.0.0 oauthlib-3.0.1 pyasn1-0.4.5 pyasn1-modules-0.2.5 python-dateutil-2.8.0 pyyaml-5.1 requests-2.21.0 requests-oauthlib-1.2.0 rsa-4.0 six-1.12.0 urllib3-1.25.2 websocket-client-0.56.0

```

Thank you, regards

### PR DESCRIPTION
**Pull request by aogier**
fixes #829 

thank you, regards

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ certifi>=14.05.14 # MPL
 six>=1.9.0  # MIT
 python-dateutil>=2.5.3  # BSD
 setuptools>=21.0.0  # PSF/ZPL
-urllib3>=1.23  # MIT
+urllib3>=1.23,<1.25  # MIT
```
Comment (micw523) [inline requirements.txt]:
```suggestion
urllib3==1.24.3  # MIT
```

**Comment by micw523** [comment]
Also, your CLA seems to be problematic since your commit shows "a authored and b committed" the change.

File: requirements.txt (hunk 1 above)
Comment (aogier) [reply requirements.txt]: lgtm

**Comment by aogier** [comment]
> Also, your CLA seems to be problematic since your commit shows "a authored and b committed" the change.

yes, that's because I repushed my commit under an already signed account for CLA, if this is not acceptable I'll then wait for eidosmedia to approve contributions and then I'll resign under proper mail

**Comment by micw523** [comment]
1) If you're part of the organization then the organization also needs to sign the CLA. 
2) When you authored the commit, your git profile should contain your GitHub email address in it.
I'm not an expert in dealing with Git though so I could not provide further help on this...

**Comment by micw523** [comment]
https://help.github.com/en/articles/why-are-my-commits-linked-to-the-wrong-user
This may help but I'm not sure.

**Comment by roycaihw** [comment]
Closing in favor of #897

/close

**Pull request closed without merging**